utils/navigator_gui/navigator_gui/host_monitor.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+'''
+Host Monitor: A simple node to monitor and publish the IP address status of the
+core hosts on the platform's network.
+'''
+
+
+import socket
+import subprocess
+
+from navigator_msgs.msg import Hosts
+import rospy
+
+
+__author__ = "Anthony Olive"
+__maintainer__ = "Anthony Olive"
+__email__ = "anthony@iris-systems.net"
+__copyright__ = "Copyright 2016, MIL"
+__license__ = "MIT"
+
+
+rospy.init_node("host_monitor")
+
+
+class HostMonitor():
+
+    def __init__(self):
+        self.pub_hosts = rospy.Publisher("/host_monitor", Hosts, queue_size=1)
+
+        # Build an ordered list of host dictionaries that resolve to devices on navigator
+        host_list = [
+            "mil-nav-ubnt-wamv",
```

Comment:
What machines do these hostnames map to? There are three for wamv. It would also be nice to add the IP of the Ubiquity on both sides, as it would be very helpful for debugging network outages.